src/cloud/components/experiments/DatalessEmptyState.tsx

```diff
@@ -0,0 +1,78 @@
+// Libraries
+import React, {FC} from 'react'
+import {useHistory} from 'react-router-dom'
+import {connect, ConnectedProps} from 'react-redux'
+
+// Components
+import {
+  EmptyState,
+  ComponentSize,
+  ComponentColor,
+  Button,
+} from '@influxdata/clockface'
+
+// Utils
+import {getOrg} from 'src/organizations/selectors'
+import {getAll} from 'src/resources/selectors'
+
+// Types
+import {AppState, Bucket, ResourceType} from 'src/types'
+
+interface OwnProps {
+  children: JSX.Element
+}
+
+type ReduxProps = ConnectedProps<typeof connector>
+
+type Props = OwnProps & ReduxProps
+
+const DatalessEmptyState: FC<Props> = ({orgID, buckets, children}) => {
+  const history = useHistory()
+
+  const handleClientLibrary = (): void => {
+    history.push(`/orgs/${orgID}/load-data/sources`)
+  }
+
+  // TODO: Replace with data loading flag from api if experiment is successful
+  if (buckets.find(bucket => bucket.type === 'user')) {
```

Comment:
What about the demo data bucket "Website Monitoring" - is that one of type 'user'? I honestly don't know

Reply:
Ok I checked - "Website Monitoring" demo bucket also has `.type = 'user'` so this logic is good 👍

Reply:
Demo data buckets are of type `demoData` however this does remind me that we automatically create a bucket for the user when they sign up and that initial bucket is of type `user` so this is a problem.

Reply:
Hmm interesting because when I looked at the API response in cloud2 for buckets, it looks like the type is set to `user`. But it's probably good to handle the other cases too. 
![image](https://user-images.githubusercontent.com/6411855/106198662-2bd87b00-6169-11eb-8c5e-c8f3172ca3cf.png)

Reply:
Oh, that's interesting mine says type: "demoData". Seems this isn't a reliable check. I'll need to find a better way to approach this.
![image](https://user-images.githubusercontent.com/11937365/106204429-cccb3400-6171-11eb-924c-d67556abb888.png)

Reply:
I've modified this now to get buckets of type "user" and check their cardinality to see if there's data in them or not.

Reply:
So my concern now is that we're going to potentially charge the user every time this component is mounted for a query poking at each of there buckets 😢

Reply:
This check will stop querying buckets immediately when it finds one with data so this would only be true if they had many buckets created that were all empty. I have, however, added an additional guard to skip the query and assume they have data if they have more than 4 buckets created.